crates/router/src/core/payments/operations/payment_confirm.rs

```diff
@@ -410,6 +410,15 @@ impl<F: Clone> UpdateTracker<F, PaymentData<F>, api::PaymentsRequest> for Paymen
         let setup_future_usage = payment_data.payment_intent.setup_future_usage;
         let business_label = Some(payment_data.payment_intent.business_label.clone());
         let business_country = Some(payment_data.payment_intent.business_country);
+        let description = payment_data.payment_intent.description.clone();
+        let statement_descriptor_name = payment_data
+            .payment_intent
+            .statement_descriptor_name
+            .clone();
+        let statement_descriptor_suffix = payment_data
+            .payment_intent
+            .statement_descriptor_suffix
+            .clone();
```

Comment:
Since the `payment_data.payment_intent` is being overwritten anyways...
we could have done `.take()` instead of `.clone()` to save an allocation...